hw1/counter.js

```diff
@@ -0,0 +1,42 @@
+function createCounter(initialValue) {
+    const userVariable = {
+        userNumber: initialValue,
+        changeNumber: initialValue,
+    };
+
+    let {userNumber = 0, changeNumber = 0} = userVariable;
```

Comment:
I didn't really get, why do you need to create the object and then destructure it 🙂 
You could simply use next code instead (works absolutely the same, but shorter and without creating an object in memory):
```suggestion
let userNumber = initialValue;
let changeNumber = initialValue;
```